crypto/csharp/CryptoOpFpe.cs

```diff
@@ -105,8 +113,13 @@ static void Main(string[] args)
                 }
 
                 inputBytes = Encoding.ASCII.GetBytes(input);
-
-
+		
+		/* For FPE algorithm, the input bytes should be encoded using UT8 and not ASCII. */
+                // inputBytes = Encoding.UTF8.GetBytes(input);
+		
+		/* For FPE set IV only when data size is greater than its block size eg CARD10: 56, CARD26: 40, CARD62:32.
+                 * UNICODE, the block size is calculated based on Cardinality. */
+		    
                 /*Set IV only when data size is more than 56 Bytes */
                 if (inputBytes.Length > 56)
```

Comment:
Add comment like "Set IV only when data size is more than 56 Bytes in CARD10" add comment for CARD26 and CARD62